package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/ui-component-library",
-  "version": "0.7.8",
+  "version": "0.7.9",
```

Comment:
as @dblane-digicatapult would say, this is an addition/feature, should be the second number increase

```
va.b.c
a - completely new way
b - new feature added
c - hotfix
```

Reply:
Sort of. Where this get's dicey is we have a major version `0` which basically means anything goes. https://semver.org/#spec-item-4

Reply:
changed to minor bump